modules/ROOT/pages/mock-event-processor.adoc

```diff
@@ -144,16 +144,28 @@ You can assert that every time the HTTP request fails, your application returns
 </munit:test>
 ----
 <1> Mock an HTTP Requester with the configuration of the requester in your flow.
-<2> Configure a then-return element to throw an `HTTP:CONNECTIVITY` error. +
+<2> Configure a then-return element to throw an *HTTP:CONNECTIVITY* error. +
     This triggers the On-error scope in your application.
 <3> Execute the flow containing the requester.
 <4> Assert that the returned payload is the one you set inside the On-error scope.
 
-It is not possible to return all known error types of the Mule application. You can only return the type of modules used by the current flow. If you try to return an error type out of the scope of the being-tested flow, you get the `MULE:UNKNOWN` error instead of the configured one.
+You cannot return all known error types of a Mule application. You can only return the type of modules used by the current flow. If you try to return an error type that is out of the scope of the flow you're testing, you get a *MULE:UNKNOWN* error instead of the configured one.
```

Comment:
@egrave, please, can you confirm that the "module" word is appropriate in the sentence "You can only return the type of modules used by the current flow"? The previous sentence reads: "You cannot return all known error types of a Mule application". Should it read:
"You can only return the type of **_errors_** used by the current flow".
Thanks!

Reply:
I think the idea is : You can only return the type of errors  defined in the modules used by the current flow

Reply:
Thanks for the clarification!